README.md

```diff
@@ -1,24 +1,43 @@
-* [Some kind of color picker library](https://blog.logrocket.com/color-picker-libraries-to-use-in-2021/)
-* Maybe Vue because I haven't used it before? Or React because I have and I want some pro tips? (I am also really curious about using React Native if that is on the table at all--although that might be out of scope for the apprenticeship?)
-* Beyond that, open to whatever seems like it might lend itself well to this.
-### What does the game do? (additional details for decomping)
-* Allows you to generate a color grid (optionally) - a much more polished, embellished version of the CodePen
-  * You generate this color grid by picking 4 corner colors
-  * It would be really nice if there were a color picker that could be utilized to do this
-  * Then you can pick from several preset sizes for the grid, for instance, a 4x4, 8x8, 16x16 (small, medium, large)
-  * Then it creates a grid of tiles with the 4 picked colors in each corner and tiles in between that gradually change from one corner color to the other
-* ORR the grids come pre-made and you pick the one you like and the size (I like this idea a lot less and would be happy to make it just a grid generating playground as an MVP before a game)
-* The game then shows you the grid with the colors put in order, then scrambles the tiles in the grid. It is then up to the player to put them back together in the right order by switching each misplaced tile with the one that goes in its spot.
-* The player will receive some kind of feedback so that they know when the tile is in the correct place, maybe a dot or some marker on the correctly (or incorrectly) placed tile.
+## Organization
+In the `src` folder, there are a number of folders that contain JavaScript files:
+* board
+* color-utils
+* grid
+* helpers
+
+Plus the `index.js` file, which serves as the entry point file for the app. 
+### Game/Sandbox Functions Files
+`board` and `helpers` contain files for the rendering of the color grid and the playing of the game.
+
+### Grid Generator Files
+The main achievement of the app is that it can take four corner colors (top left, top right, bottom left, bottom right) and generate a color grid containing tiles colored with those four corner colors, gradually transitioning from one corner color to another in between. The code for achieving this can be found in `grid` and `color-utils`.
+
+When four corner colors are submitted in the form, `makeGrid` in `grid.js` is called, which looks at the first color and last color in each row and calls `makeRow` in `row.js` to create each row of tiles that gradually transition in color from start to end. `makeRow` converts the start and end color in each row to rgb values then uses `makeStops` to compare the r, g, and b values (numbers between 0 and 255) at start and end and determine, based on how many color tiles should be in each row, what numerical values should go between the start and end to generate evenly spaced values.
```

Comment:
There's don't think there's anything wrong with including logic in the README, thought it isn't vary commonplace (i'd say).